### PR TITLE
PMM-2334 Update golang to 1.12.1

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

File: build/bin/build-server-rpm [hunk 1]
```diff
@@ -206,8 +206,8 @@ d }\" \
                 spectool -C SOURCES -g SOURCES/${spec_name}.spec
 
                 sudo go version
-                sudo wget https://dl.google.com/go/go1.11.5.linux-amd64.tar.gz
-                sudo tar -C /usr/local -xzf go1.11.5.linux-amd64.tar.gz
+                sudo wget https://dl.google.com/go/go1.12.1.linux-amd64.tar.gz
```
Comment (adivinho) [inline build/bin/build-server-rpm]: Could you leave it as 1.11.5 ?
The latest version isn't working smoothly with grafana.

Comment (EvgeniyPatlan) [reply build/bin/build-server-rpm]: this question is not to me :) It was requested to get 1.12.1

Comment (EvgeniyPatlan) [reply build/bin/build-server-rpm]: @rnovikovP could you please check and decide

Comment (EvgeniyPatlan) [reply build/bin/build-server-rpm]: In general I think we should check latest grafana and fix qan2

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-213-f0fc275
client docker - perconalab/pmm-client-fb:PR-213-f0fc275
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm2-client/pmm2-client-PR-213-f0fc275.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-213-04df12d
client docker - perconalab/pmm-client-fb:PR-213-04df12d
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm2-client/pmm2-client-PR-213-04df12d.tar.gz

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-213-aaaff52
client docker - perconalab/pmm-client-fb:PR-213-aaaff52
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm2-client/pmm2-client-PR-213-aaaff52.tar.gz

File: build/bin/build-server-rpm (hunk 1 above)
Comment (adivinho) [reply build/bin/build-server-rpm]: The latest Grafana build has no fix for the issue.

**Comment by JNKPercona** [comment]
server docker - perconalab/pmm-server-fb:PR-213-bf209a3
client docker - perconalab/pmm-client-fb:PR-213-bf209a3
client - https://s3.us-east-2.amazonaws.com/pmm-build-cache/pmm2-client/pmm2-client-PR-213-bf209a3.tar.gz

**Pull request merged**